chapter17/dtj/ex2

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/perl
+
+use strict;
+use warnings;
+
+# Pretty sure $ENV{'PWD'} isn't portable
```

Comment:
Since it's just `PWD`, I think we can use `opendir ( DIR, '.' ) or die $!;` 

There is also an alternative of [Globbing](http://techbus.safaribooksonline.com/9781449311063/id2961572).